src/qt/clientmodel.cpp

```diff
@@ -243,7 +243,7 @@ static void BlockTipChanged(ClientModel *clientmodel, bool initialSync, int heig
         clientmodel->cachedBestHeaderTime = blockTime;
     }
     // if we are in-sync or if we notify a header update, update the UI regardless of last update time
-    if (fHeader || !initialSync || now - nLastUpdateNotification > MODEL_UPDATE_DELAY) {
+    if (fHeader && !fReindex || !initialSync || now - nLastUpdateNotification > MODEL_UPDATE_DELAY) {
```

Comment:
please put parenthesis around `fHeader && !fReindex` to make it unambiguous to readers if that's what you meant

Reply:
ACK